### PR TITLE
add JH HashMap (UniqueWordCounter)

### PR DESCRIPTION
**Pull request by Sarmat92**

### CODE REVIEW

File: forJIS7/mvnw [hunk 1]
```diff
@@ -54,7 +54,7 @@ case "`uname`" in
   CYGWIN*) cygwin=true ;;
   MINGW*) mingw=true;;
   Darwin*) darwin=true
-    # Use /usr/libexec/java_home if available, otherwise fall back to /Library/Java/Home
+    # Use /usr/libexec/java_home if available, otherwise fall back to /BookLibrary/Java/Home
```
Comment (ArtikTico) [inline forJIS7/mvnw:57]: .gitignore

File: forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Override
+    public void showAllProduct() {
+        for (Product productEach:getProductList()) {
+            System.out.println(productEach);
+        }
+    }
+
+    @Override
+    public void fiendProductByName(String productName) {
```
Comment (ArtikTico) [inline forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java]: `findProductByName` you have syntax mistake.
And if method start with `find` I expected that this method get me some objects, in your cases I want get Product. 
But you return void

File: forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        boolean isFound = false;
+        for (Product productEach:getProductList()) {
+            if (productEach.getProductName().equals(productName)){
+                System.out.println(productEach.getProductName() + " " +productEach.getProductCost());
+                isFound = true;
+            }
+        }
+        if (!isFound){
```
Comment (ArtikTico) [inline forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java]: rewrite method without flag `true` and `false`

**Review by ArtikTico — COMMENTED** [review]
Check my comments

File: forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java (hunk 2 above)
Comment (Sarmat92) [reply forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java]: fixed

File: forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java (hunk 3 above)
Comment (Sarmat92) [reply forJIS7/src/main/java/JavaHomework/Lectures_10/Collections/Shop/Service/ShopServiceImplementation.java]: done

**Pull request merged**